components/filter/README.md

```diff
@@ -220,12 +221,43 @@ This component is built to be used alongside the [d2l-filter-dimension-set](#d2l
 
 | Property | Type | Description |
 |---|---|---|
+| `count` | Number | Count for the value in the list. If no count is provided, no count will be displayed |
 | `disabled` | Boolean, default: `false` | Whether the value in the filter is disabled or not |
 | `key` | String, required | Unique identifier within a dimension for the value |
 | `text` | String, required | Text for the value in the list |
 | `selected` | Boolean, default: `false` | Whether the value in the filter is selected or not |
 <!-- docs: end hidden content -->
 
+## Counts [d2l-filter-dimension-set-value]
+
+The `count` property displays a count next to each filter value to indicate the number of results a value will yield. This helps users more effectively explore data and make selections, so it’s a good idea to provide these counts if it can be done performantly.
+
+Note that when using multiple filter dimensions, the counts should be updated when selections are made across dimensions so that they always reflect the number of results a filter will yield.
+
+<!-- docs: demo live align:start autoOpen:true autoSize:false size:large -->
```

Comment:
You probably want to be using `demo code` here instead of `demo live`. `demo live` displays a web component's properties, slots and events in addition to the code and demo. Though it may be displaying correctly at the moment since it was not given a web component `name`. [See here](http://localhost:8000/about/adding-component/#readme%2Fmarkdown-document)